jenkins/Makefile

```diff
@@ -22,8 +22,7 @@ RPMBUILD_BASEIMAGE_DEFAULTS = \
 	fedora-38 fedora-37 fedora-36 \
 	centos-9 centos-8 centos-7
 
-# All Fedora images are based on Fedora 36
```

Comment:
We might need to change this comment as well.

Reply:
Actually...I deleted that comment.  🙂  Is that a sufficient change?  😀